libs/sql-schema-describer/src/mssql.rs

```diff
@@ -248,9 +262,9 @@ impl SqlSchemaDescriber {
                             .or_else(|| DEFAULT_STRING.captures_iter(&default_string).next())
                             .or_else(|| DEFAULT_DB_GEN.captures_iter(&default_string).next())
                             .map(|cap| cap[1].to_string())
-                            .expect(&format!("Couldn't parse default value: `{}`", default_string));
+                            .expect("Couldn't parse default value");
```

Comment:
can you restore this please, we introduced this to diagnose errors.

Reply:
Ah thanks for spotting this one. ✅